data/faq.html

```diff
@@ -1062,6 +1264,29 @@ <h3 lang="es">Búsqueda</h3>
 <p></p>
 Nota: solamente puedes encontrar los sitios que han sido mapeados en 
 <a href="http://www.openstreetmap.org">OpenStreetMap</a>.</dd>
+<dd lang="fr">Cliquer sur l'icone de recherche dans l'écran principal. Vous pouvez rechercher en
+utilisant l'un des cas suivants :
+<p></p>
+<b><em>Adresse</em></b>: entrez le N° de la maison, le nom de la rue. Vous pouvez également inquer
+le nom de la ville (si vous ne trouvez
+toujours pas) et le code postal. Par exemple: 41, Rue des Alouettes 75000 Paris.
+<p></p>
+<b><em>Nom d'un lieu</em></b> : Par exemple : Place de la Mairie.
+<p></p>
+<b><em>Mots clefs</em></b>. Les catégories les plus populaires sont visibles par des icones dans
```

Comment:
I guess semicolon also should be used here. Or a dot for all items in this list.